README.md

```diff
@@ -5,6 +5,33 @@ This is a HACS custom integration for enphase envoys with firmware version 7.X.
 1. Install [HACS](https://hacs.xyz/) if you haven't already
 2. Add this repository as a [custom integration repository](https://hacs.xyz/docs/faq/custom_repositories) in HACS
 4. Restart home assistant
-5. Add the integration through the home assistant configuration flow
+5. Add the integration through the home assistant configuration flow. Look for "Enphase Envoy (DEV)" .
+6. Click "Use Englighten" and use your enlighten username and password in the fields
+6. Add integral sensors if you want to use the energy dashboard. This is nessesary because Homeassistant requires the dashboard import/export sensors to *not* count energy you instantly produced and used yourself. Therefore, we need to integrate the power sensors Envoy does provides. Be nice if this could be in the integration, but not implimented yet. You can also use the GUI to make these now: Settings->Device and Services->Helpers. But yaml is easier! XXXXXX is your serial number. You then use `envoy_XXXXXX_grid_export_energy` and `envoy_XXXXXX_grid_import_energy` in the energy dashboard, along with `envoy_XXXXXX_daily_production` (which the integration provides directly. Can also use lifetime if prefer).Add below into your configuration.yaml (if you already have a sensor block, just add these platforms into it)
+```
+sensor:
+  - platform: integration
+    source: sensor.envoy_XXXXXX_current_grid_export_power
+    name: envoy_XXXXXX_grid_export_energy
+    unit_prefix: k
+    round: 2
+  - platform: integration
+    source: sensor.envoy_XXXXXX_current_grid_import_power
+    name: envoy_XXXXXX_grid_import_energy
+    unit_prefix: k
+    round: 2
+```
```

Comment:
This should all be in a separate section and marked as optional. Please also add very explicit statements that this will not be accurate for battery installations.